src/cascade/executor/usage_reporter.py

```diff
@@ -0,0 +1,44 @@
+import os
+import json
+
+from cascade.core.db import cursor
+
+"""
+CREATE TABLE epi.dismod_resource_usage (
+    run_id varchar(32),
+    jobid varchar(255),
+    job_key text,
+    process_key text,
+    scale_parameters text,
+    inserted_at datetime,
+    summary text
+)
+"""
```

Comment:
What is this stray quotation up to?

Reply:
Oh. Hmm. I put that there to keep track of my creation statement because we don't have an established way of tracking schema state. I don't really want to set that up for this one table but I feel weird not having a record of the schema somewhere. I don't know. I didn't mean to leave this orphaned like it is but I'm not sure where it belongs.

Reply:
Oh, right. When I have a system over which I have control, I make management functions or management command-line scripts. create_tables(), destroy_tables(), archive_tables(). Maybe put it into a create_tables() command?